bundles/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/handler/BlindHandler.java

```diff
@@ -0,0 +1,184 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.lutron.internal.handler;
+
+import static org.openhab.binding.lutron.internal.LutronBindingConstants.*;
+
+import java.math.BigDecimal;
+
+import org.eclipse.smarthome.core.library.types.PercentType;
+import org.eclipse.smarthome.core.library.types.StopMoveType;
+import org.eclipse.smarthome.core.library.types.UpDownType;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.RefreshType;
+import org.openhab.binding.lutron.internal.config.BlindConfig;
+import org.openhab.binding.lutron.internal.protocol.LutronCommandType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Handler responsible for communicating with Lutron blinds
+ *
+ * @author Bob Adair - Initial contribution based on Alan Tong's DimmerHandler
+ */
+public class BlindHandler extends LutronHandler {
+    private static final Integer ACTION_LIFTLEVEL = 1;
```

Comment:
These are only used as string. Wouldn't it be better to just declare them as `String`?

Reply:
Probably. This code was derived from ShadeHandler, which was derived from DimmerHandler, and this is the way it was done there. :-) I'll take a look at it.

Reply:
I took a look at this, and these protocol constants probably should be changed to strings everywhere. I have another update that I'm working on to support light dimming down to 0.1% as well as sending/receiving other command parameters that have up to two digits after the decimal. (See #6287). This will require going through all the LIP command send and receive code paths to make the appropriate changes, and then re-testing everything afterward. So I'd rather make these changes at the same time I do that.